ingeniamotion/wizard_tests/feedbacks_tests/feedback_test.py

```diff
@@ -407,14 +407,15 @@ def loop(self) -> ResultType:
     def generate_output(
         self, position_displacement: float, negative_displacement: float
     ) -> ResultType:
-        test_output = 0
-
-        test_output += self.check_symmetry(position_displacement, negative_displacement)
-        test_output += self.check_resolution(position_displacement)
-        test_output = self.ResultType.SUCCESS if test_output == 0 else test_output
+        symmetry_check_result = self.check_symmetry(position_displacement, negative_displacement)
+        if symmetry_check_result != self.ResultType.SUCCESS.value:
+            return self.ResultType.SYMMETRY_ERROR
+        resolution_check_result = self.check_resolution(position_displacement)
+        if resolution_check_result != self.ResultType.SUCCESS.value:
+            return self.ResultType.RESOLUTION_ERROR
         polarity = self.check_polarity(position_displacement)
         self.suggest_polarity(polarity)
-        return self.ResultType(test_output)
+        return self.ResultType.SUCCESS
 
     def get_result_msg(self, output: ResultType) -> str:
```

Comment:
get_result_msg was made to concatenate errors if more than one happens, but now it makes no sense. Please, simplify it.

Reply:
Done